crates/ruma-common/src/api.rs

```diff
@@ -318,6 +318,9 @@ pub enum SendAccessToken<'a> {
     /// Always add the access token.
     Always(&'a str),
 
+    /// Only add the token for appservices.
```

Comment:
It is not clear what "only … for appservices" means.

Maybe something like:

```suggestion
    /// Add the given appservice token to the request only if the `METADATA` on the request requires
    /// it.
```